### PR TITLE
[AKS] BREAKING CHANGE: `az aks trustedaccess rolebinding create`: Remove deprecated `-r` and `-s` options

### PR DESCRIPTION
**Pull request by YitongFeng-git**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command

az aks trustedaccess rolebinding create

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Review by karataliu — APPROVED** [review]
LGTM

File: src/aks-preview/HISTORY.rst [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 Release History
 ===============
+* [BREAKING CHANGE]: `az aks trustedaccess rolebinding create`: Remove deprecated `-r` and `-s` options
```
Comment (FumingZhang) [inline src/aks-preview/HISTORY.rst:5]: Please move this paragraph down a few lines and choose a new version number with increased major version.

Comment (FumingZhang) [reply src/aks-preview/HISTORY.rst:5]: And also update the value of `VERSION` in `setup.py`.

Comment (FumingZhang) [reply src/aks-preview/HISTORY.rst:5]: Note, there's another breaking change #8217

Comment (YitongFeng-git) [reply src/aks-preview/HISTORY.rst]: Thanks! updated!

**Review by FumingZhang — APPROVED** [review]
LGTM

**Pull request merged**